src/lib/core/media-marshaller/media-marshaller.ts

```diff
@@ -152,10 +160,12 @@ export class MediaMarshaller {
   /** Track element value changes for a specific key */
   trackValue(element: HTMLElement, key: string): Observable<ElementMatcher> {
     return this.subject.asObservable()
-      .pipe(filter(v => v.element === element && v.key === key));
+        .pipe(filter(v => v.element === element && v.key === key));
   }
 
-  /** update all styles for all elements on the current breakpoint */
+  /**
+   * update all styles for all elements on the current breakpoint
```

Comment:
Undo style change

Reply:
why?

Reply:
To be consistent with the rest of the lib, single-line comments are on a literal single line.